### PR TITLE
Apply Unrouted label to validated issues

### LINKED ISSUE #44: Apply a `Status: Unrouted` label to all issues that pass validation
### Problem

The Open Source Team is responsible for routing all incoming issues to the appropriate team, but we don't have a good queue to work against, which makes it hard to ensure that we're not missing issues. The general GitHub notification stream is waaaaay too noisy (regardless of how it's consumed: email, web, etc.).

### Solution

Teach the `validate-new-issue` action to apply a `Status: Unrouted` label to issues that pass validation. With this label humans (for now) can use a search like this to stay on top of routing:

https://github.com/search?q=org%3Agetsentry+label%3A%22Status%3A+Unrouted%22+state%3Aopen&type=Issues

### PR DESCRIPTION
**Pull request by chadwhitacre**
Closes #44.

### CODE REVIEW

**Comment by chadwhitacre** [comment]
Green CI: https://github.com/getsentry/.github-chadwhitacre/actions/runs/573045382.

File: .github/workflows/validate-new-issue-test.yml [hunk 1]
```diff
@@ -192,3 +192,34 @@ jobs:
...
+          ### Environment
+          ### Steps to Reproduce
+          ### Expected Result
+          ### Actual Result
+          "))
+          echo "::set-output name=issue_number::${issue_number}"
+      - name: test
+        run: |
+          issue_number="${{ steps.setup.outputs.issue_number }}"
+          .github/workflows/test-helpers/get-issue-validation-log "${issue_number}"
```
Comment (BYK) [inline .github/workflows/validate-new-issue-test.yml:216]: So we are still using debug logs to test things? I mean this one should be pretty simple, just check whether the issue has that label or not?
```shell
gh issues view $issue_number | grep "^Labels:"
``

should come close, no?

**Review by BYK — APPROVED** [review]
Nice :)

File: .github/workflows/validate-new-issue-test.yml (hunk 1 above)
Comment (chadwhitacre) [reply .github/workflows/validate-new-issue-test.yml:216]: > So we are still using debug logs to test things?

Not really. Imagine that `get-issue-validation-log` is renamed to `wait-for-issue`, as it is in fact renamed on #46. :)

> gh issues view $issue_number  | grep "^Labels:"

See comment on #46 at https://github.com/getsentry/.github/pull/46/files#diff-300cc84d45c813ed2972f3ea7f5e093e0f99dda06427a2b70bfac39c0a2d0470R7-R9. I had trouble using `gh issues view` in a pipeline, hence the reliance on JSON output there and here.

**Pull request merged**